pkg/export/debug/debug.go

```diff
@@ -52,6 +125,29 @@ func printFunc() (pipe.FinalFunc[[]request.Span], error) {
 	}, nil
 }
 
+func serializeSpansJSON(spans []request.Span, indent bool) ([]byte, error) {
+	if indent {
+		return json.MarshalIndent(spans, "", " ")
+	}
+
+	return json.Marshal(spans)
+}
+
+func jsonPrintFunc(indent bool) (pipe.FinalFunc[[]request.Span], error) {
+	return func(input <-chan []request.Span) {
+		for spans := range input {
+			data, err := serializeSpansJSON(spans, indent)
+
+			if err != nil {
+				fmt.Printf("Error serializing json\n")
```

Comment:
If this error printing code should remain, we probably want to use slog().

Reply:
Will do